src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
+const currentHealth = 100;
```

Comment:
```suggestion
const DEFAULT_HEALTH = 100;
```
It's default I think, and it should be in upper case